zoomage_report.CURATED.tsv

```diff
@@ -55815,3 +55815,81 @@ sampling site	macula lutea	http://purl.obolibrary.org/obo/UBERON_0000053	E-GEOD-
 sampling site	macula lutea	http://purl.obolibrary.org/obo/UBERON_0000053	E-GEOD-137537	SAMN12771947	Nancy George	13/08/2020 16:00
 sampling site	sinoatrial node	http://purl.obolibrary.org/obo/UBERON_0002351	E-HCAD-16	SAMN12046033	Nancy George	13/08/2020 16:00
 specimen with known storage state	fresh	http://purl.obolibrary.org/obo/OBI_0000971	E-HCAD-7	group2	Nancy George	13/08/2020 16:00
+organism part	airway submucosal gland	http://www.ebi.ac.uk/efo/EFO_0010666			Nancy George	28/08/2020 16:00
+organism part	alveolar sac	http://purl.obolibrary.org/obo/UBERON_0002169			Nancy George	28/08/2020 16:00
+organism part	body of pancreas	http://purl.obolibrary.org/obo/UBERON_0001150			Nancy George	28/08/2020 16:00
+organism part	central vein of liver	http://purl.obolibrary.org/obo/UBERON_0006841			Nancy George	28/08/2020 16:00
+cell type	central venous hepatic sinusoidal endothelial cell	http://www.ebi.ac.uk/efo/EFO_0010706			Nancy George	28/08/2020 16:00
+organism part	chorionic plate	http://purl.obolibrary.org/obo/UBERON_0004027			Nancy George	28/08/2020 16:00
+organism part	chorionic villus	http://purl.obolibrary.org/obo/UBERON_0007106			Nancy George	28/08/2020 16:00
+cell type	Clara cell	http://purl.obolibrary.org/obo/CL_0000158			Nancy George	28/08/2020 16:00
```

Comment:
This should be "club cell", no?

Reply:
I see it has it as synonym. So should be ok.

Reply:
changed it anyway 👍